config.template/faf-rabbitmq/enabled_plugins

```diff
@@ -0,0 +1 @@
+[rabbitmq_management,rabbitmq_prometheus].
```

Comment:
Why do we need `rabbitmq_management`?

Reply:
rabbitmq_management is the web dashboard. It is currently active and would be overwritten by that change.